simplepso/pso.py

```diff
@@ -247,27 +245,6 @@ def set_bounds(self, parameter_range=None, lower=None, upper=None):
         self.ub = upper
         self.bounds_set = True
 
-    def _calc_fitness_from_array(self, traj, num_sim):
-
-        for i, part in enumerate(self.population):
-            start = i * num_sim
-            end = (i + 1) * num_sim
-            tmp_results = traj[list(range(start, end))]
-            tmp_results = tmp_results.T.unstack('simulation')
-            error = self.cost_function(tmp_results)
-            part.fitness = error
-            self.population_fitness[i] = error
-
-    def _get_parameters_from_population(self, num_sims, total_sims, n_params):
-        """ Create param_array for GPU based simulators """
-        rate_vals = np.zeros((total_sims, n_params))
-        # create parameters for each particle, creates blocks per num_sims
-        for i, part in enumerate(self.population):
-            start = i * num_sims
-            end = (i + 1) * num_sims
-            rate_vals[start:end, :] = part.pos
-        return rate_vals
-
     def run(self, num_particles, num_iterations, num_processes=1,
```

Comment:
In pysb the argument to set the number of cores is named: `num_processors`, would it be good to keep the same name for simplePSO?

Reply:
Good idea! Changed.